catalyst-gateway/bin/src/service/api/v1/mod.rs

```diff
@@ -29,4 +34,39 @@ impl V1Api {
     ) -> account_votes_get::AllResponses {
         account_votes_get::endpoint(state, account_id).await
     }
+
+    /// Process fragments
+    ///
+    /// Posts a fragments batch to be processed.
+    #[oai(
+        path = "/fragments",
+        method = "post",
+        operation_id = "fragments",
+        tag = "ApiTags::Fragments"
+    )]
+    async fn fragments_post(
```

Comment:
Both endpoints need to be marked deprecated.
As per the Epic `they are to be marked as deprecated to signal our intent to obsolete them.`